app.py

```diff
@@ -92,8 +93,9 @@ class BatteryConfig:
 
 @dataclass
 class TimeSeriesData:
-    gt: List[float]  # Required total energy
-    ft: List[float]  # Forecasted production
+    dt: List[float] # time step length
```

Comment:
```suggestion
    dt: List[int] # time step length
```

Don't need floats here???

Reply:
I'm not sure if floats come with additional costs here (is float in python larger than int?), on the first occasion they get involved in a float operation anyway which will cause an implicit type cast. Or is that different in python?

Reply:
Keine Ahnung. Aber da daraus auch das Api generiert wird fände ich int hier angemessener?